packages/flutter/lib/src/material/app.dart

```diff
@@ -208,9 +207,16 @@ class MaterialApp extends StatefulWidget {
   /// message.
   final RouteFactory onUnknownRoute;
 
-  /// Callback that is called when the operating system changes the
-  /// current locale.
-  final LocaleChangedCallback onLocaleChanged;
+  /// The initial locale for this app's [Localizations] widget.
+  ///
+  /// If the 'locale' is null the system's locale value is used.
```

Comment:
It's arguments (that don't have matching properties) for which we avoid [foo]. (For those we use backticks.)